stdlib/Makefile

```diff
@@ -0,0 +1,34 @@
+default: get
+
+PKG3_VERSION := f4139648bfb4cdde34e50947a73f71c7ee8d29a5
+PKG3_URL := https://github.com/vchuravy/Pkg3.jl
```

Comment:
Is this actually where it's going to live?

Reply:
No this is just where the changes are to make it actually work right now.